### PR TITLE
[sc-59665] v0.37 changelog

### PR DESCRIPTION
**Pull request by isaacdevlugt**
https://app.shortcut.com/xanaduai/objective/59665

### CODE REVIEW

File: doc/releases/changelog-0.37.0.md [hunk 1]
```diff
@@ -103,75 +103,79 @@
 * `default.clifford` now supports arbitrary state-based measurements with `qml.Snapshot`.
   [(#5794)](https://github.com/PennyLaneAI/pennylane/pull/5794)
 
-* Implemented kwargs (`check_interface`, `check_trainability`, `rtol` and `atol`) support in `qml.equal` for the operators `Pow`, `Adjoint`, `Exp`, and `SProd`.
-  [(#5668)](https://github.com/PennyLaneAI/pennylane/issues/5668)
+* `qml.equal` now properly handles `Pow`, `Adjoint`, `Exp`, and `SProd` operators as arugments across
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:106]:
```suggestion
* `qml.equal` now properly handles `Pow`, `Adjoint`, `Exp`, and `SProd` operators as arguments across 
```

File: doc/releases/changelog-0.37.0.md [hunk 2]
```diff
@@ -103,75 +103,79 @@
...
 
-* The `default.qubit` device implements a depth-first tree-traversal algorithm to
-  accelerate native mid-circuit measurement execution. The new implementation
-  supports classical control, collecting statistics, and post-selection, along
-  with all measurements enabled with `qml.dynamic_one_shot`.
+* The `default.qubit` device supports a depth-first tree-traversal algorithm to accelerate native mid-circuit 
+  measurement execution. The new implementation supports classical control, collecting statistics, and 
+  post-selection, along with all measurements enabled with `qml.dynamic_one_shot`. More information 
+  about this new mid-circuit measurement method can be found on our 
+  [measurement documentation page](https://docs.pennylane.ai/en/stable/introduction/measurements.html).
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:140]:
```suggestion
  [measurement documentation page](https://docs.pennylane.ai/en/stable/introduction/dynamic_quantum_circuits.html#tree-traversal-algorithm).
```

File: doc/releases/changelog-0.37.0.md [hunk 3]
```diff
@@ -103,75 +103,79 @@
...
 
-<h4>Faster and flexible mid-circuit measurements</h4>
+<h4>Faster and more flexible mid-circuit measurements</h4>
 
-* The `default.qubit` device implements a depth-first tree-traversal algorithm to
-  accelerate native mid-circuit measurement execution. The new implementation
-  supports classical control, collecting statistics, and post-selection, along
-  with all measurements enabled with `qml.dynamic_one_shot`.
+* The `default.qubit` device supports a depth-first tree-traversal algorithm to accelerate native mid-circuit 
+  measurement execution. The new implementation supports classical control, collecting statistics, and
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:137]:
```suggestion
  measurement execution. Accessible through the QNode argument `mcm_method="tree-traversal"`,
  this new implementation supports classical control, collecting statistics, and 
```

File: doc/releases/changelog-0.37.0.md [hunk 4]
```diff
@@ -103,75 +103,79 @@
...
+  * `dynamic_one_shot` now uses a single auxiliary tape with a shot vector and `default.qubit` implements 
+    the loop over shots with `jax.vmap`.
+    [(#5617)](https://github.com/PennyLaneAI/pennylane/pull/5617)
+  * `dynamic_one_shot` is now compatible with `jax.jit`.
+    [(#5557)](https://github.com/PennyLaneAI/pennylane/pull/5557)
+
+* Mid-circuit measurement tests have been streamlined and refactored, removing most end-to-end tests 
+  from the native MCM test file, but keeping one that validates multiple mid-circuit measurements with 
+  any allowed return and interface end-to-end tests.
   [(#5787)](https://github.com/PennyLaneAI/pennylane/pull/5787)
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:173]: Minor, but this feels not to exciting and maybe worth moving to the bottom of this section.

File: doc/releases/changelog-0.37.0.md [hunk 5]
```diff
@@ -242,84 +244,79 @@
...
   [(#5523)](https://github.com/PennyLaneAI/pennylane/pull/5523)
   [(#5686)](https://github.com/PennyLaneAI/pennylane/pull/5686)
   [(#5889)](https://github.com/PennyLaneAI/pennylane/pull/5889)
 
-* The `qml.pytrees` module now has `flatten` and `unflatten` methods for serializing pytrees.
+* The `qml.PyTrees` module now has `flatten` and `unflatten` methods for serializing PyTrees.
   [(#5701)](https://github.com/PennyLaneAI/pennylane/pull/5701)
 
-* `qml.sample` can now be used on Boolean values representing mid-circuit measurement results in
+* `qml.sample` can now be used on boolean values representing mid-circuit measurement results in
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:270]: I think it is normally capitalized

File: doc/releases/changelog-0.37.0.md [hunk 6]
```diff
@@ -242,84 +244,79 @@
...
-* Add support for 3 new pytest markers: `unit`, `integration` and `system`.
+* Three new Pytest markers have been added for easier management of our test suite: `unit`, `integration` 
+  and `system`.
   [(#5517)](https://github.com/PennyLaneAI/pennylane/pull/5517)
 
 <h4>Other improvements</h4>
 
-* `expectation_value` was added to `qml.math` to calculate the expectation value of a matrix for pure states.
+* `qml.MultiControlledX` can now be decomposed even when no `work_wires` are provided. The implementation 
+  returns $\mathcal{O}(\text{len(control\_wires)}^2)$ operations and is applicable for any multi-controlled
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:314]:
```suggestion
  returns :math:`\mathcal{O}(\text{len(control\_wires)}^2)` operations and is applicable for any multi-controlled 
```
Doesn't currently show up correctly in the [docs](https://xanaduai-pennylane--5928.com.readthedocs.build/en/5928/development/release_notes.html)

File: doc/releases/changelog-0.37.0.md (hunk 5 above)
Comment (isaacdevlugt) [reply doc/releases/changelog-0.37.0.md:270]:
```suggestion
* `qml.sample` can now be used on Boolean values representing mid-circuit measurement results in
```

File: doc/releases/changelog-0.37.0.md [hunk 7]
```diff
@@ -6,17 +6,79 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
 
-* The `default.tensor` device now supports the `tn` method to simulate quantum circuits using exact tensor networks.
+* A new `default.tensor` device is now available for performing
+  [tensor network](https://en.wikipedia.org/wiki/Tensor_network) and
+  [matrix product state](https://en.wikipedia.org/wiki/Matrix_product_state) simulations using the
+  [`quimb` backend](https://quimb.readthedocs.io/en/latest/).
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:12]:
```suggestion
  [quimb backend](https://quimb.readthedocs.io/en/latest/).
```

File: doc/releases/changelog-0.37.0.md [hunk 8]
```diff
@@ -6,17 +6,80 @@
...
+  With this bond dimension, the expectation values calculated for `default.qubit` and
+  `default.tensor` are different:
+
+  ```pycon
+  >>> np.abs(value_dq - value_mps)
+  tensor(0.0253213, requires_grad=True)
+  ```
+
+  Learn more about `default.tensor` and how to configure it by visiting the
+  [how-to guide](https://pennylane.ai/qml/demos/tutorial_How_to_simulate_quantum_circuits_with_tensor_networks/).
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:82]:
```suggestion
  Learn more about `default.tensor` and how to configure it by 
  [visiting the how-to guide](https://pennylane.ai/qml/demos/tutorial_How_to_simulate_quantum_circuits_with_tensor_networks/).
```

File: doc/releases/changelog-0.37.0.md [hunk 9]
```diff
@@ -6,17 +6,80 @@
...
+      for i in range(num_qubits - 1):
+          qml.CNOT([i, i + 1])
+      return qml.expval(qml.Z(num_qubits - 1))
+
+  circuit()
+  ```
+
+  Additional configuration settings like the matrix product state bond dimension can be specified
+  when instantiating `default.tensor`, see the
+  [device documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_tensor.DefaultTensor.html)
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:41]:
```suggestion
  [device's documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_tensor.DefaultTensor.html)
```

**Review by trbromley — COMMENTED** [review]
This will maybe fix the docs build errors/warnings.

File: doc/releases/changelog-0.37.0.md [hunk 10]
```diff
@@ -6,57 +6,171 @@
...
+  value_mps = qml.QNode(f, dev_mps)()
+  ```
+
+  With this bond dimension, the expectation values calculated for `default.qubit` and
+  `default.tensor` are different:
+
+  ```pycon
+  >>> np.abs(value_dq - value_mps)
+  tensor(0.0253213, requires_grad=True)
+  ```
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.37.0.md:82]: Should we add some timing results here as well?

File: doc/releases/changelog-0.37.0.md [hunk 11]
```diff
@@ -6,57 +6,171 @@
 
 <h4>Execute faster with Default Tensor 🔗</h4>
```
Comment (Qottmann) [inline doc/releases/changelog-0.37.0.md:7]:
```suggestion
<h4>Execute bigger with Default Tensor 🔗</h4>
```

I would suggest this reframing since the main point of the device is size and not speed. In fact, most <20q circuits are running slower on default.tensor than default.qubit, as the point is more scalability (at this point at least)

File: doc/releases/changelog-0.37.0.md [hunk 12]
```diff
@@ -6,57 +6,171 @@
...
+  [(#5744)](https://github.com/PennyLaneAI/pennylane/pull/5744)
   [(#5786)](https://github.com/PennyLaneAI/pennylane/pull/5786)
+  [(#5795)](https://github.com/PennyLaneAI/pennylane/pull/5795)
 
-* The `default.tensor` device is introduced to perform tensor network simulations of quantum circuits using the `mps` (Matrix Product State) method.
-  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  Either method can be selected when instantiating the `default.tensor` device by setting the
+  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
+  device can simulate a large number of qubits with sensible approximations. Take this example that
+  can simulate 1000 qubits in a few seconds!
```
Comment (Qottmann) [inline doc/releases/changelog-0.37.0.md:22]:
```suggestion
  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
  device can simulate a large number of qubits with low-to-intermediate entanglement. Take this example that
  can simulate 1000 qubits with a shallow circuit in a few seconds!
```

The full contraction method (in its current form) is always exact and there is no approximation. Both methods have different strengths, but it would be correct to generally say for both that they are good for simulating low-to-intermediately entangled circuits. "Low-to-intermediate" is a bit of a mouthful an vague, could be replaced by something else in case you dont like it :)

File: doc/releases/changelog-0.37.0.md [hunk 13]
```diff
@@ -6,57 +6,171 @@
...
+  >>> circuit()
+  tensor(0., requires_grad=True)
+  ```
+
+  Additional configuration settings like the matrix product state bond dimension can be specified
+  when instantiating `default.tensor` (see the
+  [device's documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_tensor.DefaultTensor.html)
+  for more details).
+  These settings have implications for the speed and accuracy of the simulation, as shown in the
+  following example. First, set up the circuit:
```
Comment (Qottmann) [inline doc/releases/changelog-0.37.0.md:48]:
```suggestion
  For matrix product state simulations (`method="mps"`), we can make the exeuction approximate by setting `max_bond_dim` (see the
  [device's documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_tensor.DefaultTensor.html)
  for more details).
  The maximum bond dimension lets us control the degree of the approximation and has implications for the speed of the simulation, as shown in the
  following example. First, set up the circuit:
```

Some details as the default for `max_bond_dim` is such that the execution is always exact.

**Review by Qottmann — COMMENTED** [review]
Looks great @isaacdevlugt ! 🥳 

Just some minor tweaks as suggestions on the terminology for the default.tensor part :)

File: doc/releases/changelog-0.37.0.md (hunk 11 above)
Comment (isaacdevlugt) [reply doc/releases/changelog-0.37.0.md:7]: Fair point! Wasn't aware of DT being slower than DQ for <20 qubits 🤔... Is this strictly implementation that's causing this? I would expect that I could at least get super fast results with a crappy bond dimension for upwards of ~12 qubits, no?

File: doc/releases/changelog-0.37.0.md (hunk 12 above)
Comment (isaacdevlugt) [reply doc/releases/changelog-0.37.0.md:22]: This is always a tough thing to describe without going into too much detail, so the nuance I left out with what's currently written is probably confusing you — totally understandable 😅. The point I'm making is akin to saying "trotterization is efficient with sensible approximations". The implementation of the method under the hood in PennyLane is exact, but certainly the trotterization technique is approximating something. 

So, with MPS, the implementation of the contraction method is exact, and anything under the sun can be represented by an MPS... _with a big enough bond dimension_, and that's what I'm talking about when I say "approximations". If a crappy bond dimension is chosen, the result is an approximation of the exact result. 

(_takes breath_)

What's probably better to say here is this (leave even more specifics out 😆, and let the user discover the wonderful world of matrix product states)

```suggestion
  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
  device can efficiently simulate a large number of qubits. Take this example that
  can simulate 1000 qubits in a few seconds!
```

The reason I don't love "low-to-intermediate entanglement" here is because you can have lots of entanglement so long as we have area-law scaling. 

TLDR; let's just be super vague 😆

File: doc/releases/changelog-0.37.0.md (hunk 11 above)
Comment (Qottmann) [reply doc/releases/changelog-0.37.0.md:7]: Yeah, the devil is in the detail: So for "typical" circuits you have to go to larger qubit sizes to get a chance to beat default.qubit, even for small bond dimensions. There are some known performance bottlenecks from re-canonicalizing the MPS state that we havent addressed yet, plus overall there are some competing overheads from SVDs that make it slower.

Check this example with 8 qubits. Even with trivial bond dimension of 2 DQ is faster:
```
dataset = qml.data.load("qchem", molname="H4", bondlength=0.92, basis="STO-3G")[0]

H = dataset.hamiltonian
n_wires = len(H.wires)

def circuit():
    qml.BasisState(dataset.hf_state, wires=H.wires)
    for op in dataset.vqe_gates:
        qml.apply(op)
    return qml.expval(H)

bond_dims = 2**np.arange(2, (n_wires//2)+1) # maximum required bond dinemsion is 2**(n_wires//2) = 64
ress = []

bond_dim = 2
mps = qml.device("default.tensor", wires=n_wires, method="mps", max_bond_dim=bond_dim, contract="nonlocal")
dev = qml.device("default.qubit")
qnode_mps = qml.QNode(circuit, mps)
qnode_def = qml.QNode(circuit, dev)

%timeit res = qnode_mps()
179 ms ± 5.34 ms per loop (mean ± std. dev. of 7 runs, 1 loop each)
%timeit res = qnode_def()
116 ms ± 3.49 ms per loop (mean ± std. dev. of 7 runs, 10 loops each)
```

This is one example of how performance can look like (here the circuit depth is constant)
image
The take away is that typically the break-even point is somewhere between 12-20 qubits, depending on the circuit and bond dimension. So people switching out the device for small examples will likely first see a performance regression (before they scale up qubit count)

File: doc/releases/changelog-0.37.0.md (hunk 12 above)
Comment (Qottmann) [reply doc/releases/changelog-0.37.0.md:22]: I get your point about the MPS method, just want to clarify that the full contraction method (`method="tn"`) really is exact-exact and there is (currently) no way to introduce approximations.

File: doc/releases/changelog-0.37.0.md (hunk 11 above)
Comment (isaacdevlugt) [reply doc/releases/changelog-0.37.0.md:7]:
```suggestion
<h4>Execute wide circuits with Default Tensor 🔗</h4>
```

File: doc/releases/changelog-0.37.0.md (hunk 12 above)
Comment (isaacdevlugt) [reply doc/releases/changelog-0.37.0.md:22]: Oooooooooh okay okay... apologies 😆 I over explained the crap out of my perspective then lol. In that case, I still prefer my suggestion above. You good with that? (emphasis on the "can" in "can efficiently simulate ...")

File: doc/releases/changelog-0.37.0.md [hunk 14]
```diff
@@ -4,59 +4,172 @@
...
+  [(#5744)](https://github.com/PennyLaneAI/pennylane/pull/5744)
   [(#5786)](https://github.com/PennyLaneAI/pennylane/pull/5786)
+  [(#5795)](https://github.com/PennyLaneAI/pennylane/pull/5795)
 
-* The `default.tensor` device is introduced to perform tensor network simulations of quantum circuits using the `mps` (Matrix Product State) method.
-  [(#5699)](https://github.com/PennyLaneAI/pennylane/pull/5699)
+  Either method can be selected when instantiating the `default.tensor` device by setting the
+  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
+  device can simulate a large number of qubits with sensible approximations. Take this example that
+  can simulate 1000 qubits in a few seconds!
```
Comment (Qottmann) [inline doc/releases/changelog-0.37.0.md:22]:
```suggestion
  Either method can be selected when instantiating the `default.tensor` device by setting the
  `method` keyword argument to `"tn"` (tensor network) or `"mps"` (matrix product state). This
  device can simulate a large number of qubits. Take this example that
  simulates 1000 qubits with a shallow circuit in a few seconds!
```

perhaps like this?
Claiming that it can _efficiently_ simulate a large number of qubits without saying any caveats could be misinterpreted I think

Comment (isaacdevlugt) [reply doc/releases/changelog-0.37.0.md:22]: :shipit:

File: doc/releases/changelog-0.37.0.md [hunk 15]
```diff
@@ -4,175 +4,422 @@
...
+
+  ```pycon
+  >>> circuit()
+  tensor(0., requires_grad=True)
+  ```
+
+  For matrix product state simulations (`method="mps"`), we can make the exeuction approximate by setting `max_bond_dim` (see the
+  [device's documentation](https://docs.pennylane.ai/en/stable/code/api/pennylane.devices.default_tensor.DefaultTensor.html)
+  for more details).
+  The maximum bond dimension lets us control the degree of the approximation and has implications for the speed of the simulation, as shown in the
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.37.0.md:46]: Just rewording a bit because we don't show any timing results

```suggestion
  The maximum bond dimension has implications for the speed of the simulation and lets us control the degree of the approximation, as shown in the
```

File: doc/releases/changelog-0.37.0.md [hunk 16]
```diff
@@ -4,175 +4,422 @@
...
+      qml.breakpoint()
+
+      return qml.sample()
+
+  circuit(1.2345)
+  ```
+
+  Upon executing `script.py`, the simulation pauses at the first breakpoint:
+
+
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.37.0.md:206]:
```suggestion
```

File: doc/releases/changelog-0.37.0.md [hunk 17]
```diff
@@ -4,175 +4,422 @@
...
+  
+  Finally, to modify a circuit mid-run, simply call the desired PennyLane operations:
+
+  ```pycon
+  (pldb) qml.CNOT(wires=(0,2))
+  CNOT(wires=[0, 2])
+  (pldb): print(qml.debug_tape().draw(wire_order=[0,1,2]))
+  0: ──H─╭●─╭●─┤  
+  2: ────╰X─╰X─┤  
+  ```
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.37.0.md:246]: We have a demo coming soon, right?

```suggestion
  ```

Stay tuned for an in-depth demonstration on using this feature in real-world examples!
```

Comment (isaacdevlugt) [inline doc/releases/changelog-0.37.0.md:246]: We have a demo coming soon, right?

```suggestion

Stay tuned for an in-depth demonstration on using this feature with real-world examples!
```

Comment (isaacdevlugt) [inline doc/releases/changelog-0.37.0.md:246]: We have a demo coming soon, right?

```suggestion

Stay tuned for an in-depth demonstration on using this feature with real-world examples!
```

File: doc/releases/changelog-0.37.0.md [hunk 18]
```diff
@@ -4,175 +4,422 @@
...
+      qml.BasicEntanglerLayers(x, range(n_wires))
+      return qml.expval(qml.X(0))
+  ```
+
+  ```pycon
+  >>> print(qml.draw(f, level="device")())
+  0: ──RX(0.28)─╭●────╭X──RX(0.70)─╭●────╭X─┤  <X>
+  1: ──RX(0.52)─╰X─╭●─│───RX(0.65)─╰X─╭●─│──┤     
+  2: ──RX(0.00)────╰X─╰●──RX(0.03)────╰X─╰●─┤     
+  ```
```
Comment (isaacdevlugt) [inline doc/releases/changelog-0.37.0.md:345]: Nice!

File: doc/releases/changelog-0.37.0.md [hunk 19]
```diff
@@ -4,178 +4,424 @@
...
+          qml.CNOT([i, i + 1])
+      return qml.expval(qml.Z(num_qubits - 1))
+  ```
+
+  ```pycon
+  >>> circuit()
+  tensor(0., requires_grad=True)
+  ```
+
+  For matrix product state simulations (`method="mps"`), we can make the exeuction approximate by setting `max_bond_dim` (see the
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:43]:
```suggestion
  For matrix product state simulations (`method="mps"`), we can make the execution be approximate by setting `max_bond_dim` (see the
```

File: doc/releases/changelog-0.37.0.md [hunk 20]
```diff
@@ -4,178 +4,424 @@
...
-* A new `qml.noise` module which contains utility function for building `NoiseModels` 
-  and an `add_noise` tranform for addding it to quantum circuits.
+* Support for building noise models and applying them to a quantum circuit has been added
+via the `NoiseModel` class and an `add_noise` transform.
   [(#5674)](https://github.com/PennyLaneAI/pennylane/pull/5674)
   [(#5684)](https://github.com/PennyLaneAI/pennylane/pull/5684)
   [(#5718)](https://github.com/PennyLaneAI/pennylane/pull/5718)
 
+  Under the hood, PennyLane's approach to noise models is insertion-based, meaning that noise is included
+  by *inserting* additional operators (gates or channels) that describe the noise into the quantum circuit itself.
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:95]:
```suggestion
  by *inserting* additional operators (gates or channels) that describe the noise into the quantum circuit.
```

File: doc/releases/changelog-0.37.0.md [hunk 21]
```diff
@@ -4,178 +4,424 @@
...
+  * `qml.noise.wires_in(wires)`: if an operator is applied to any wire in `wires`, add noise.
+  * custom noise conditions: custom conditions can be defined as functions decorated with `qml.BooleanFn` that return a Boolean value. For example, the following function will insert noise if a `qml.RY` operator is encountered with an angle of rotation that is less than `0.5`:
+
+    ```python
+    @qml.BooleanFn
+    def c0(op):
+        return isinstance(op, qml.RY) and op.parameters[0] < 0.5
+    ```
+
+  `op_eq`, `op_in`, `wires_eq`, and `wires_in` can also be combined together with `&`, `and`, `|`, etc.
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:111]:
```suggestion
  Conditions can also be combined together with `&`, `and`, `|`, etc.
```

File: doc/releases/changelog-0.37.0.md [hunk 22]
```diff
@@ -4,178 +4,424 @@
...
   [(#5881)](https://github.com/PennyLaneAI/pennylane/pull/5881)
+  
+  For fermionic operators:
 
-  ```python
-  of_op = openfermion.FermionOperator('0^ 2')
-  pl_op = qml.from_openfermion(of_op)
-  of_op_new = qml.to_openfermion(pl_op)
+  ```pycon
+  >>> of_fermionic = openfermion.FermionOperator('0^ 2')
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:257]:
```suggestion
  >>> import openfermion
  >>> of_fermionic = openfermion.FermionOperator('0^ 2')
```

**Review by trbromley — APPROVED** [review]
A few more comments from checking the code.

File: doc/releases/changelog-0.37.0.md [hunk 23]
```diff
@@ -4,178 +4,425 @@
...
 * `default.clifford` now supports arbitrary state-based measurements with `qml.Snapshot`.
   [(#5794)](https://github.com/PennyLaneAI/pennylane/pull/5794)
 
+* `qml.equal` now properly handles `Pow`, `Adjoint`, `Exp`, and `SProd` operators as arguments across 
+  different interfaces and tolerances with the addition of four new keyword arguments: `check_interface`, 
+  `check_trainability`, `atol` and `rtol`.
+  [(#5668)](https://github.com/PennyLaneAI/pennylane/pull/5668)
+  
 * The sorting order of parameter-shift terms is now guaranteed to resolve ties in the absolute value with the sign of the shifts.
   [(#5583)](https://github.com/PennyLaneAI/pennylane/pull/5583)
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:360]:
```suggestion
```
Not a community contrib

File: doc/releases/changelog-0.37.0.md [hunk 24]
```diff
@@ -360,60 +607,68 @@
...
   [(#5791)](https://github.com/PennyLaneAI/pennylane/pull/5791)
 
-* Transforms applied to callables now use `functools.wraps` to preserve the docstring and call signature of the original function.
+* Transforms applied to callables now use `functools.wraps` to preserve the docstring and call signature 
+  of the original function.
   [(#5857)](https://github.com/PennyLaneAI/pennylane/pull/5857)
 
-* `qml.qsvt()` now supports jax arrays with angle conversions. 
+* `qml.qsvt()` now supports JAX arrays with angle conversions. 
   [(#5853)](https://github.com/PennyLaneAI/pennylane/pull/5853)
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:659]:
```suggestion

* The sorting order of parameter-shift terms is now guaranteed to resolve ties in the absolute value with the sign of the shifts.
  [(#5583)](https://github.com/PennyLaneAI/pennylane/pull/5583)

```

File: doc/releases/changelog-0.37.0.md [hunk 25]
```diff
@@ -274,56 +526,49 @@
...
-* Add support for 3 new pytest markers: `unit`, `integration` and `system`.
+* Three new Pytest markers have been added for easier management of our test suite: `unit`, `integration` 
+  and `system`.
   [(#5517)](https://github.com/PennyLaneAI/pennylane/pull/5517)
 
 <h4>Other improvements</h4>
 
-* `expectation_value` was added to `qml.math` to calculate the expectation value of a matrix for pure states.
+* `qml.MultiControlledX` can now be decomposed even when no `work_wires` are provided. The implementation 
+  returns :math:`\mathcal{O}(\text{len(control\_wires)}^2)` operations and is applicable for any multi-controlled
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:566]:
```suggestion
  returns :math:`\mathcal{O}(\mbox{len(control\_wires)}^2)` operations and is applicable for any multi-controlled 
```

File: doc/releases/changelog-0.37.0.md [hunk 26]
```diff
@@ -339,16 +584,18 @@
 * `qml.TrotterProduct` is now compatible with resource tracking by inheriting from `ResourcesOperation`. 
    [(#5680)](https://github.com/PennyLaneAI/pennylane/pull/5680)
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:585]:
```suggestion
  [(#5680)](https://github.com/PennyLaneAI/pennylane/pull/5680)
```

File: doc/releases/changelog-0.37.0.md [hunk 27]
```diff
@@ -274,56 +524,49 @@
...
-* Add support for 3 new pytest markers: `unit`, `integration` and `system`.
+* Three new Pytest markers have been added for easier management of our test suite: `unit`, `integration` 
+  and `system`.
   [(#5517)](https://github.com/PennyLaneAI/pennylane/pull/5517)
 
 <h4>Other improvements</h4>
 
-* `expectation_value` was added to `qml.math` to calculate the expectation value of a matrix for pure states.
+* `qml.MultiControlledX` can now be decomposed even when no `work_wires` are provided. The implementation 
+  returns :math:`\mathcal{O}(\mbox{len(control\_wires)}^2)` operations and is applicable for any multi-controlled
```
Comment (trbromley) [inline doc/releases/changelog-0.37.0.md:564]:
```suggestion
  returns :math:`\mathcal{O}(\mbox{len(control wires)}^2)` operations and is applicable for any multi-controlled 
```
